subgraphs/aave-v2-forks-ext/protocols/aave-v2/config/templates/aave.v2.template.yaml

```diff
@@ -4,11 +4,11 @@ schema:
 dataSources:
   - kind: ethereum/contract
     name: LendingPoolAddressesProvider
-    network: {{ network }}
+    network: { { network } }
```

Comment:
you might want to revert this change, assuming it's done by prettier

Reply:
Hmm - Yeah. This might be an issue. This was triggered because I merged with the master branch. Ideally this wouldn't trigger prettier I'd think. Especially since it could affect deployments.

Reply:
Will want to rebase on master instead of merge.